life_expectancy/cleaning.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import pandas as pd
+from life_expectancy.config import DATA_DIR
+
+def parse_args():
+    """Parse command line arguments"""
+    parser=argparse.ArgumentParser()
+    parser.add_argument("region", help="Country code to filter the data", nargs='?', const='PT')
```

Comment:
Important: typically we'd like to use region as a flag in the unix style, please replace "region" -> "--region"